[2023-09-29T10:23:31.558+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2023-09-29T10:23:31.563+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL.transform_data scheduled__2023-09-29T00:00:00+00:00 [queued]>
[2023-09-29T10:23:31.567+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL.transform_data scheduled__2023-09-29T00:00:00+00:00 [queued]>
[2023-09-29T10:23:31.567+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-09-29T10:23:31.573+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): transform_data> on 2023-09-29 00:00:00+00:00
[2023-09-29T10:23:31.577+0000] {standard_task_runner.py:57} INFO - Started process 173 to run task
[2023-09-29T10:23:31.580+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL', 'transform_data', 'scheduled__2023-09-29T00:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpe4783okg']
[2023-09-29T10:23:31.581+0000] {standard_task_runner.py:85} INFO - Job 15: Subtask transform_data
[2023-09-29T10:23:31.590+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-09-29T10:23:31.614+0000] {task_command.py:415} INFO - Running <TaskInstance: ETL.transform_data scheduled__2023-09-29T00:00:00+00:00 [running]> on host 2651bdac0f0e
[2023-09-29T10:23:31.620+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2023-09-29T10:23:31.646+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2023-09-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-29T00:00:00+00:00'
[2023-09-29T10:23:31.647+0000] {logging_mixin.py:151} INFO - handle data 
[2023-09-29T10:23:31.647+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-09-29T10:23:31.650+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL, task_id=transform_data, execution_date=20230929T000000, start_date=20230929T102331, end_date=20230929T102331
[2023-09-29T10:23:31.672+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-29T10:23:31.680+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
